R/OsModel.R

```diff
@@ -0,0 +1,118 @@
+#' Title - TODO
+#'
+#' Description - TODO
+#'
+#' @slot stan TODO
+#' @slot templated Logical. Defines whether the OsModel object needs to be completed
+#' @export
+OsModel <- setClass(
+    Class = "OsModel",
+    representation = list(
```

Comment:
```suggestion
    slots = list(
```
See `?setClass`: "All these arguments are deprecated from version 3.0.0 of R and should be avoided."